source/puppet/4.4/reference/environments_configuring.markdown

```diff
@@ -1,6 +1,7 @@
 ---
 layout: default
 title: "Configuring Environments"
+canonical: "/puppet/latest/reference/environments_configuring.html"
```

Comment:
This is [now auto-discovered](https://confluence.puppetlabs.com/display/DOCS/YAML+Frontmatter#YAMLFrontmatter-canonical:), so we don't need to explicitly write it.